packages/forms-web-app/src/lib/appeals-api-wrapper.js

```diff
@@ -268,8 +229,66 @@ exports.errorMessages = {
 	}
 };
 
+/**
+ * V2 endpoints
+ */
+
 exports.getUserAppealsById = async (id) => {
 	return handler(`/api/v2/users/${id}/appeals`, 'GET');
+}
```

Comment:
We should move this to the new `appeals-api-client` in common - same with `updateUser`.